components/accelerator/nvidia/query/infiniband/default.go

```diff
@@ -32,9 +32,9 @@ var gpuPortConfigs = map[string]ExpectedPortStates{
 	// ref. https://docs.nvidia.com/dgx/dgxb200-user-guide/introduction-to-dgxb200.html
 	"b200": {AtLeastPorts: 8, AtLeastRate: 400},
 
-	// "provides 1.8 terabytes per second (TB/s) of GPU-to-GPU interconnect, InfiniBand networking"
-	// ref. https://www.nvidia.com/en-us/data-center/gb200-nvl72/
-	"gb200": {AtLeastPorts: 8, AtLeastRate: 1800},
+	// GB200 NVL2 supports "NVIDIA Quantum-X800 InfiniBand" of 800 GB/s
+	// ref. https://www.nvidia.com/en-us/data-center/gb200-nvl2/
+	"gb200": {AtLeastPorts: 8, AtLeastRate: 800},
```

Comment:
at least should be 400. there are two versions... 400 and 800.

Reply:
connectx-8 is the 800gb version ->https://nvdam.widen.net/s/hbp8zz7fvt/solution-overview-gtcspring24-quantum-x800-3175164

connectx-7 is 400gb

Reply:
>Performance and Throughput:

> ConnectX-7: Offers up to 400Gb/s of throughput with up to four ports of connectivity. It delivers hardware-accelerated networking, storage, security, and manageability services suitable for cloud, telecommunications, AI, and enterprise workloads. 
NVIDIA.COM

> ConnectX-8 SuperNIC: Designed to meet the massive data demands of AI workloads, it delivers up to 800Gb/s throughput for unmatched speed and efficiency.

Reply:
Yeah checking the references

Reply:
fixed